Перетасовка органов дион

ErdGinalD opened the pull request:
Перетасовка органов дион по всему телу согласно пройденному авоуту:

https://discord.com/channels/617003227182792704/755125334097133628/1064577756047888384

Теперь грудь содержит глаза, мозг и лёгкие, а пах содержит уши, сердце, почки, печень и аппендикс. Голова полностью пустая, что не снижает эффективности дионы вне аварийных ситуаций при её потере.

ErdGinalD committed:
Чето высрал не понял нахуя

выглядит тоже как говно

ErdGinalD committed:
Исправление ошибок

Я только познаю табуляцию и уже оттабулировать того кто это придумал

ErdGinalD committed:
ПРОБЕЛ!

еще фикс

Rerik007 commented on code/modules/surgery/organs/eyes.dm:
```diff
@@ -79,3 +79,9 @@
 	origin_tech = "biotech=4"
 	status = ORGAN_ROBOT
+
+/obj/item/organ/internal/ears/diona
```
Почему эти уши задаются тут, а не в organs/subtypes/diona.dm?

Rerik007 commented on code/modules/mob/living/carbon/human/species/diona.dm:
```diff
@@ -36,15 +36,17 @@
+		"receptor node" =   /obj/item/organ/internal/eyes/diona, //Default darksight of 2.
+		"gas bladder" =   /obj/item/organ/internal/lungs/diona,
+		"polyp segment" =   /obj/item/organ/internal/appendix/diona,
+		"anchoring ligament" = /obj/item/organ/internal/heart/diona,
+		"oscillatory catcher" = /obj/item/organ/internal/ears/diona
```
```suggestion
		"anchoring ligament" = /obj/item/organ/internal/heart/diona
```
Уши меняются в mutantears. Тут путать будет. Их можно спокойно убрать отсюда.

ErdGinalD committed:
Изменение формулы фторсерной кислоты

Изменение формулы фторсерной кислоты и количеству оной в крапиве согласно авоуту:
https://discord.com/channels/617003227182792704/755125334097133628/1061712987137769512

ErdGinalD committed:
Update code/modules/mob/living/carbon/human/species/diona.dm

Co-authored-by: Rerik007 <120549107+Rerik007@users.noreply.github.com>

ErdGinalD committed:
я не понял как удалить так что переизменил ошибки

ErdGinalD committed:
и еще

Rerik007 commented on code/modules/surgery/organs/eyes.dm:
```diff
@@ -79,3 +79,9 @@
 	desc = "An electronic device designed to mimic the functions of a pair of human eyes. It has no benefits over organic eyes, but is easy to produce."
 	origin_tech = "biotech=4"
 	status = ORGAN_ROBOT
+
+/obj/item/organ/internal/ears/diona
+	name = "oscillatory catcher"
+	icon = 'icons/mob/alien.dmi'
+	icon_state = "claw"
+	desc = "A strange organic object used by a Gestalt for orientation in a three-dimensional projection"
```
```suggestion

```
Перенести вниз. В organs/subtypes/diona.dm, где должны задаваться органы. Я предложил как.

Rerik007 commented on code/modules/surgery/organs/subtypes/diona.dm:
```diff
@@ -81,51 +81,57 @@
+/obj/item/organ/internal/ears/diona
+	species_type = /datum/species/diona
+	name = "oscillatory catcher"
+	parent_organ = "groin"
```
```suggestion
	icon = 'icons/mob/alien.dmi'
	icon_state = "claw"
	desc = "A strange organic object used by a Gestalt for orientation in a three-dimensional projection."
	parent_organ = "groin"
```
Перенести с верха сюда. Расовые органы должны быть в таких файлах.

ErdGinalD committed:
Update code/modules/surgery/organs/eyes.dm

Co-authored-by: Rerik007 <120549107+Rerik007@users.noreply.github.com>

ErdGinalD committed:
Update code/modules/surgery/organs/subtypes/diona.dm

Co-authored-by: Rerik007 <120549107+Rerik007@users.noreply.github.com>
